elide-contrib/elide-dynamic-config-helpers/src/main/resources/elideTableSchema.json

```diff
@@ -0,0 +1,296 @@
+  "$id": "http://elide.io/schemas/table_schema_v1#",
+  "description": "Elide Table config json/hjson schema",
+  "definitions": {
+    "join": {
+      "title": "Join",
+      "description": "Joins describe the SQL expression necessary to join two physical tables. Joins can be used when defining dimension columns that reference other tables.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "title": "Join name",
+          "description": "The name of the join relationship.",
+          "type": "string"
+        },
+        "to": {
+          "title": "Join table name",
+          "description": "The name of the table that is being joined to",
+          "type": "string",
+          "pattern": "^([A-Za-z]*_?[A-Za-z]*)*$"
+        },
+        "type": {
+          "title": "Type of Join",
+          "description": "Type of the join - toOne or toMany",
+          "type": "string",
+          "enum": [
+            "toOne",
+            "toMany"
+          ]
+        },
+        "definition": {
+          "title": "Join definition SQL",
+          "description": "Templated SQL expression that represents the ON clause of the join",
+          "type": "string"
+        }
+      },
+      "required": [
+        "name",
+        "to",
+        "type",
+        "definition"
+      ]
+    },
+    "measure": {
+      "title": "Measure",
+      "description": "Metric definitions are extensible objects that contain a type field and one or more additional attributes. Each type is tied to logic in Elide that generates a metric function.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "title": "Metric name",
+          "description": "The name of the metric. This will be the same as the POJO field name.",
+          "type": "string",
+          "pattern": "^([A-Za-z]*_?[A-Za-z]*)*$"
+        },
+        "description": {
+          "title": "Metric description",
+          "description": "A long description of the metric.",
+          "type": "string",
+          "default": "[$name]"
+        },
+        "category": {
+          "title": "Measure group category",
+          "description": "Category for grouping",
+          "type": "string"
+        },
+        "hidden": {
+          "title": "Hide/Show measure",
+          "description": "Whether this metric is exposed via API metadata",
+          "type": "boolean",
+          "default": false
+        },
+        "readAccess": {
+          "title": "Measure read access",
+          "description": "Read permission for the metric.",
+          "type": "string",
+          "default": "Allow All"
+        },
+        "definition": {
+          "title": "Metric definition",
+          "description": "The definition of the metric",
+          "type": "string"
+        },
+        "type": {
+          "title": "Field type",
+          "description": "The data type of the field",
+          "type": "string",
+          "enum": [
+            "time",
```

Comment:
This needs to stay in sync with:
https://github.com/yahoo/elide/issues/1005

Reply:
The values in #1005 do not have number type. Number was specified as default in https://github.com/yahoo/elide/issues/1178. What should be the default for the new types. thanks

Reply:
Integer should be used as default